docs/dev/style-guide.rst

```diff
@@ -39,20 +39,37 @@ Content
 * Do not break the content across multiple lines at 80 characters,
   but rather break them on semantic meaning (e.g. periods or commas).
   Read more about this `here <https://rhodesmill.org/brandon/2012/one-sentence-per-line/>`_.
-* If you are cross-referencing to a different page within our website,
+* If you are :ref:`cross-referencing <style-guide:Cross-references>` to a different page within our website,
   use the ``doc`` role and not a hyperlink.
-* If you are cross-referencing to a section within our website,
+* If you are :ref:`cross-referencing <style-guide:Cross-references>` to a section within our website,
   use the ``ref`` role with the label from the `autosectionlabel extension <http://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html>`__.
+* Use ``<abstract concept>`` and ``<variable>`` as placeholders in code and URLs. For instance:
+
+  * ``https://<slug>.readthedocs.io``
+  * ``:guilabel:`<your username>` dropdown``
+* Make sure that **all bullet list items end with a period**, and don't mix periods with no periods.
 
 Word List
 ---------
 
 We have a specific way that we write common words:
 
 * ``Git repository`` for the place that stores Git repos. We used to use ``VCS``, but this is deprecated.
+* ``Git provider`` for generic references to GitHub/Bitbucket/GitLab/Gitea etc.
+  We avoid "host" and "platform" because they are slightly more ambiguous.
 * ``open source`` should be lower case, unless you are definitely referring to `OSI's Open Source Definition <https://opensource.org/osd>`.
 * ``Git`` should be upper case, except when referring to the command, when it should be written as `:program:\`git\``.
 
+Substitutions
+-------------
+
+The following substitutions are used in our documentation to guarantee consistency and make it easy to apply future changes.
+
+* ``|org_brand|`` is used for mentioning of |org_brand|
+* ``|com_brand|`` is used for mentioning of |com_brand|
+* ``|git_providers_and|`` is used to mention currently support Git providers with "and". Example: |git_providers_and|
+* ``|git_providers_or|`` is used to mention currently support Git providers with "or". Example: |git_providers_or|
+
```

Comment:
New section added based on @humitos' [comment](https://github.com/readthedocs/readthedocs.org/pull/9840#issuecomment-1364538348)

I also added two types of substitutions for supported Git providers. They are used quite a lot, so I would expect this to be very applicable in the current refactor. Examples in #9834

Reply:
I was suggesting to add a section, similar to "Cross-references" that explains the `.. seealso::`, to explain how we are going to use the `.. note::` to differentiate one platform from another.

Reply:
That missed me by a mile :) Is this point more generally about how we use admonition boxes? Like "if you want to point out a difference between org and com, use ``.. note::`` etc?

Reply:
Yes and no 🙃 . Let me expand. I wasn't saying to explain about the usage of admonitions _in general_; but the _specific case_ about using `.. note::` to differentiate community from commercial.

Reply:
Okay, I'll have to find a location for it in the style guide.